.eslintrc.js

```diff
@@ -38,7 +38,8 @@ const config = deepMerge(defaultConfig, {
     {
       files: ['examples/**/*.js'],
       rules: {
-        'import/no-unresolved': 0
+        'import/no-unresolved': 0,
+        'react/no-did-mount-set-state': 0
```

Comment:
I am not a fan of adding global overrides. If we can't find another a way around this, please use local `// eslint-disable-next-line react/no-did-mount-set-state` in the offending places.

Reply:
Done